src/Boost.sol

```diff
@@ -43,11 +40,13 @@ contract Boost is IBoost, EIP712, Ownable, ERC721URIStorage {
     mapping(address => uint256) public tokenFeeBalances;
 
     // Constant eth protocol fee (in wei) that must be paid by all boost creators
-    uint256 public ethFee;
+    uint128 public ethFee;
+
+    // The id of the next boost to be minted
+    uint64 public nextBoostId;
```

Comment:
did you mean 128 here? 

not sure its necessarily worth packing these state vars as that will just affect the deployment cost and will actually adds to read costs also

Reply:
additionally oz uses u256 for ids so we are doing an implicit cast when we call `_safeMint(_owner, boostId);`

Reply:
You're right unnecessary optimizations :) 
205ae576f65327591bfcaa8da1a85f809551806e